switch-configuration/config/scripts/README.md

```diff
@@ -0,0 +1,29 @@
+# Switch Configuration Scripts
```

Comment:
I appreciate the docs! I think this will provide a much better understanding to those who might not be familiar with the switch-config scripts